core/src/main/scala/org/apache/spark/h2o/converters/H2ODataFrame.scala

```diff
@@ -92,31 +94,27 @@ class H2ODataFrame[T <: water.fvec.Frame](@transient val frame: T,
         expectedTypes,
         partIndex)
 
+      /*a sequence of converters, per column*/
+      private val columnConverters = filteredTypes map converterCtx.get
+
+
+      private def readRow: InternalRow = {
+          val optionalData: Seq[Option[Any]] =
```

Comment:
Comparing to which version?

Thanks,
-Vlad

On Mon, Aug 29, 2016 at 6:02 PM, Michal Malohlava <notifications@github.com>
wrote:

> In core/src/main/scala/org/apache/spark/h2o/converters/H2ODataFrame.scala
> <https://github.com/h2oai/sparkling-water/pull/63#discussion_r76715733>:
>
> > @@ -92,31 +94,27 @@ class H2ODataFrame[T <: water.fvec.Frame](@transient val frame: T,
> >          expectedTypes,
> >          partIndex)
> >
> > +      /*a sequence of converters, per column*/
> > +      private val columnConverters = filteredTypes map converterCtx.get
> > +
> > +
> > +      private def readRow: InternalRow = {
> > +          val optionalData: Seq[Option[Any]] =
>
> I would be interested in comparing performance of this code - will create
> JIRA for that
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/h2oai/sparkling-water/pull/63/files/70f80f84960fdd184d016e79caeaf93627d7c422#r76715733>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAOuK22cPKGo24YAjdlQ5ncSt-xT2CRDks5qk4EJgaJpZM4Ju0JZ>
> .
>